uiServices/homepage_test.go

```diff
@@ -0,0 +1,43 @@
+package services
+
+import (
+	"github.com/go-chi/chi"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestStartUiModeSuccesfully(t *testing.T) {
+
+	r := chi.NewRouter()
```

Comment:
for future tests with lots of test scenarios, look at using for loops and t.Run

https://blog.golang.org/subtests